api/src/main/java/keywhiz/api/CreateGroupRequest.java

```diff
@@ -30,22 +30,29 @@
   @JsonProperty
   public final String description;
 
+  @Nullable
+  @JsonProperty
+  public final String app;
+
   public CreateGroupRequest(@JsonProperty("name") String name,
```

Comment:
This is the change that most concerns me--it's a change to the constructor, and I wasn't able to figure out how to have two constructors one of which had only name and description (Json threw errors).  Is it worth forcing CreateGroupRequests to have an empty app name/adding a "setAppName" method separate from the constructor to avoid this?